.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Undo this change, setup a global gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore